programs/connected/Cargo.toml

```diff
@@ -0,0 +1,23 @@
+[package]
+name = "connected"
+version = "0.1.0"
+description = "Created with Anchor"
```

Comment:
Can we set description what this program represent? For then matter of current PoC I don't think we need to optimize the organization but we might want to move it in a example section or smth to separate from actual protocol program